components/console/introduction.rst

```diff
@@ -148,23 +151,26 @@ It is possible to define your own styles using the class
     // ...
     $style = new OutputFormatterStyle('red', 'yellow', array('bold', 'blink'));
     $output->getFormatter()->setStyle('fire', $style);
-    $output->writeln('<fire>foo</fire>');
+    $output->writeln('<fire>foo</>');
 
 Available foreground and background colors are: ``black``, ``red``, ``green``,
 ``yellow``, ``blue``, ``magenta``, ``cyan`` and ``white``.
 
-And available options are: ``bold``, ``underscore``, ``blink``, ``reverse`` and ``conceal``.
+And available options are: ``bold``, ``underscore``, ``blink``, ``reverse``
+(enables the "reverse video" mode where the background and foreground colors
+are swapped) and ``conceal`` (the text will be hidden; commonly used to ask
+the user to type sensitive information).
```

Comment:
I don't think this is a very good explanation of conceal. It just makes the foreground color transparent. The text is still there (and visible when copy/pasting the terminal text). Besides that, the "input box" of questions can't be styled.